docs/apm/service-overview.asciidoc

```diff
@@ -2,4 +2,106 @@
 [[service-overview]]
 === Service overview
 
-Selecting a <<services,*service*>> brings you to the *Service overview*.
\ No newline at end of file
+The *Service overview* page contains a wide variety of charts and tables that provide
+visibility into how a service performs across your infrastructure. To help you isolate
+potential issues, select a <<services,*service*>> to bring you to the *Service overview* page.
+
+[discrete]
+[[service-latency]]
+=== Latency
+
+Response times for the service. You can filter the *Latency* chart to display the average,
+95th, or 99th percentile latency times for the service.
+
+[role="screenshot"]
+image::apm/images/latency.png[Service latency]
+
+[discrete]
+[[service-traffic-transactions]]
+=== Traffic and transactions
+
+The *Traffic* chart visualizes the average number of transactions per minute for the selected service.
+
+The *Transactions* table displays a list of _transaction groups_ for the
+selected service and includes the latency, traffic, error rate, and the impact for each transaction.
+Transactions that share the same name are grouped, and only one entry is displayed for each group.
+
+By default, transaction groups are sorted by _Impact_ to show the most used and slowest endpoints in your
+service. If there is a particular endpoint you are worried about, click *View transactions* to view the
+<<transactions, transactions overview>>.
+
+[role="screenshot"]
+image::apm/images/traffic-transactions.png[Traffic and transactions]
+
+[discrete]
+[[service-error-rates]]
+=== Error rate and errors
+
+The *Error rate* chart displays the average error rates relating to the service, within a specific time range.
+
+The *Errors* table provides a high-level view of each error message when it first and last occurred,
+along with the total number of occurrences. This makes it very easy to quickly see which errors affect
+your services and take actions to rectify them. To do so, click *View errors*.
+
+[role="screenshot"]
+image::apm/images/error-rate.png[Error rate and errors]
+
+[discrete]
+[[service-span-duration]]
+=== Span types average duration and dependencies
+
+The *Average duration by span type* chart visualizes each span type's average duration and helps you determine
+which spans could be slowing down transactions. The "app" label displayed under the
+chart indicates that something was happening within the application, which could signal that the
+agent does not have auto-instrumentation for whatever was happening during that time.
+
+The *Dependencies* table displays a list of downstream services or external connections relevant
+to the service at the selected time range. The table displays latency, traffic, error rate, and the impact of
+each dependency. By default, dependencies are sorted by _Impact_ to show the most used and the slowest dependency.
+If there is a particular dependency you are worried about, click *View service map* to view the related
```

Comment:
Again I'm not a fan of "worried." There's multiple use-cases where you could be investigating for different reasons, and they may or may not involve worry. 

![tenor-173001580](https://user-images.githubusercontent.com/9912/103973828-0c37cf00-5136-11eb-8ddd-bc960fc3fdc2.gif)

Reply:
Agreed. I'll update the wording.